src/Umbraco.Cms.Integrations.Crm.ActiveCampaign/Models/Dtos/ApiAccessDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+
+namespace Umbraco.Cms.Integrations.Crm.ActiveCampaign.Models.Dtos
+{
+    public class ApiAccessDto
+    {
+        private string _baseUrl { get; set; }
+
+        private string _apiKey { get; set; }
```

Comment:
```suggestion
        private readonly string _baseUrl;

        private readonly string _apiKey;
```

I think we can just use private fields here rather than properties, and they can be readonly as they are only set in the constructor.